content/millennium/r4/financial/charge-item.md

```diff
@@ -0,0 +1,111 @@
+---
+title: ChargeItem | R4 API
+---
+
+# ChargeItem
+
+* TOC
+{:toc}
+
+## Overview
+
+The resource ChargeItem describes the provision of healthcare provider products for a certain patient, therefore referring not only to the product, but containing in addition details of the provision, like date, time, amounts and participating organizations and persons.
+Main Usage of the ChargeItem is to enable the billing process and internal cost allocation.
+
+The following fields are returned if valued:
+
+* [ChargeItem id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.status){:target="_blank"}
+* [Code](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.code){:target="_blank"}
+* [Subject](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.subject){:target="_blank"}
+* [Context](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.context){:target="_blank"}
+* [occurrenceDateTime](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.occurrence_x_){:target="_blank"}
+* [Performer](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.performer){:target="_blank"}
+* [Requesting Organization](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.requestingOrganization){:target="_blank"}
+* [Quantity](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.quantity){:target="_blank"}
+* [Enterer](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.enterer){:target="_blank"}
+* [Entered Date](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.enteredDate){:target="_blank"}
+* [Reason](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason){:target="_blank"}
+* [Account](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.account){:target="_blank"}
+* [Extensions including custom attribute, description, modifier, net price, offset by, performing location, procedure, revenue code, and unit price](#extensions){:target="_blank"}
+
+## Terminology Bindings
```

Comment:
Do we need terminology bindings if we aren't mapping to any standard codes?

Reply:
Not entirely sure. I added it here thinking we might need to list the proprietary codes that we use/support.

Reply:
I think we normally document both types of bindings. i.e. https://fhir.cerner.com/millennium/r4/financial/account/#terminology-bindings

Reply:
Good to know. Thanks!